codegens/js-xhr/package.json

```diff
@@ -27,6 +27,7 @@
     "node": ">=8"
   },
   "devDependencies": {
-    "xmlhttprequest": "1.8.0"
+    "xmlhttprequest": "1.8.0",
+    "form-data": "3.0.0"
```

Comment:
Is this needed for newman tests?

Reply:
Yes. `form-data` is required by the snippet to run.